### PR TITLE
fix issues with PHP7 due to use of String annotation class, added testing of lower dependencies

### PR DESCRIPTION
**Pull request by lsmith77**

### CODE REVIEW

**Comment by WouterJ** [comment]
Needs a rebase (lots of unrelated changes in `.travis.yml`)

**Comment by lsmith77** [comment]
fixed

File: composer.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     },
     "require-dev": {
         "twig/twig": "~1.12,>=1.12.0",
-        "symfony-cmf/testing": "~1.2,>=1.2.6",
+        "symfony-cmf/testing": "dev-phpcr_odm_1_3 as 1.3.0",
```
Comment (dbu) [inline composer.json]: i guess we wait with merging for the next tag?

Comment (WouterJ) [inline composer.json]: we wait till the PR is merged and then just rely on `~1.3` as version constraint

**Pull request merged**